apps/web-client/package.json

```diff
@@ -40,7 +40,10 @@
 		"@esbuild-plugins/node-modules-polyfill": "~0.1.4",
 		"rollup-plugin-node-polyfills": "~0.2.1",
 		"autoprefixer": "~10.4.13",
-		"@types/uuid": "~9.0.0"
+		"@types/uuid": "~9.0.0",
+		"vite-plugin-pwa": "~0.14.7",
+		"@vite-pwa/sveltekit": "~0.2.2",
+		"workbox-window": "~6.5.4"
```

Comment:
Do we really need to depend on this? Since we're not using it directly [the transient dependency](https://github.com/vite-pwa/vite-plugin-pwa/blob/main/package.json#L108) via `vite-plugin-pwa` should be enough, right?

Reply:
We need it because when I try uninstalling it, I get this error
`Rollup failed to resolve import "workbox-window" from "/@vite-plugin-pwa/virtual:pwa-register"`
I tried removing node_modules and re-installing but I get the same error

Reply:
I see. I'd love to include this info into the package.json, next to that line, but comments are not allowed there unfortunately.